libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -952,21 +952,20 @@ bool AP_Arming::camera_checks(bool display_failure)
 
 bool AP_Arming::osd_checks(bool display_failure) const
 {
-#if OSD_ENABLED
+#if OSD_PARAM_ENABLED
```

Comment:
This line is the only one that is required. If you change you PR to just contain this change it will be ok.

Reply:
Looks like it is failing to build

Reply:
I think this should be
```
#if OSD_PARAM_ENABLED && OSD_ENABLED
```